.github/workflows/check-backport-changelog.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
                   # Find any changelog file that contains the Gutenberg PR link
                   gutenberg_pr_url="https://github\.com/WordPress/gutenberg/pull/${PR_NUMBER}"
-                  changelog_file=$(grep -rl "[-*] ${gutenberg_pr_url}" "${changelog_folder}" | head -n 1)
+                  changelog_file=$(grep -Erl "[-*] +${gutenberg_pr_url}" "${changelog_folder}" | head -n 1)
```

Comment:
This is important to review. The backport changelog has changed its format to adhere to the standard markdown formatting. The regex would not correctly recognized the autoformatted markdown.

I've extracted this change to https://github.com/WordPress/gutenberg/pull/68049 so it can be landed separate from all the autoformatting work here.